hardhat.config.ts

```diff
@@ -227,6 +241,22 @@ const config: HardhatUserConfig = {
           browserURL: 'https://bartio.beratrail.io'
         }
       },
+      {
+        network: 'oasSandVerse',
+        chainId: 20197,
```

Comment:
In this link, the chain id for oasis testnet is 9372. can you confirm which is correct
https://docs.sequence.xyz/solutions/chainlist/non-integrated/oasys-testnet/#rpc-urls

Reply:
the reference it taken from the docs https://docs.oasys.games/docs/staking/rpc-endpoint/1-1-rpc-endpoint